internal/cli/gitconfigs.go

```diff
@@ -0,0 +1,168 @@
+// Copyright © 2021 - 2023 SUSE LLC
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	gGitconfigAllFlag bool
+)
+
+// CmdGitconfig implements the command: epinio gitconfig
+var CmdGitconfig = &cobra.Command{
+	Use:           "gitconfig",
+	Aliases:       []string{"gitconfigs"},
+	Short:         "Epinio-controlled git configurations",
+	Long:          `Manage epinio-controlled git configurations`,
+	SilenceErrors: true,
+	SilenceUsage:  true,
```

Comment:
In other "parent" commands we don't have this flag, so the `help` will be shown if no arg is provided.

See `CmdApp` and `CmdService`:

https://github.com/epinio/epinio/blob/3863a0d523fd65bf8b7fd0c3a041d0c3fbd392eb/internal/cli/apps.go#L22-L29

https://github.com/epinio/epinio/blob/3863a0d523fd65bf8b7fd0c3a041d0c3fbd392eb/internal/cli/services.go#L24-L31

Reply:
Done with commit 8104bea6f0c8f43fd6d74351ee9084f41e1868a2